plugins/ksp/src/org/jetbrains/kotlin/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -212,6 +232,15 @@ class ResolverImpl(
         }
     }
 
+    fun resolveClassDeclaration(classDeclaration: KSClassDeclaration): ClassDescriptor? {
```

Comment:
Where is this used? If we are talking about signature rather than descriptor, I think this is needed in `mapToJvmSignature`.